client/src/components/contact-map/index.tsx

```diff
@@ -5,14 +5,13 @@ import { Container } from "../../box/absolute-box";
 import AOS from "aos";
 import "aos/dist/aos.css";
 
+const coordinates = [35.1815784, -120.7474753];
```

Comment:
Let's create a separate file in folder `config` and define some `constants.ts` file where you may put all such hard-coded values.